API_Guidelines.md

```diff
@@ -69,8 +69,84 @@ Payload
 }
 ```
 
+### GET single record using a query
+#### Request
+URL
+:   `apiHost.com/movies?title=Goodfellas`
+
+Ember Data method
+:   `queryRecord('movie', { title: 'Goodfellas' })`
+
+### Why?
+
+> To get a single record based on parameter criteria when the result is known to be one record.
+
+### Response
+
+HTTP Status
+:   200
+
+Payload
+:   
+```javascript
+{
+  "movies": [{
+    "id": 2,
+    "title": "Goodfellas",
+    "year": "1990"
+  }]
+}
+```
+
+Payload (If no data is found, then `queryRecord` returns a `null`)
+:
+```javascript
+{
+  data: null
```

Comment:
this looks jsonAPI, so i just want to confirm the RESTSerializer will return thi